suzieq/engines/pandas/ospf.py

```diff
@@ -102,6 +103,11 @@ def _get_combined_df(self, **kwargs):
         nbr_df = self._get_table_sqobj('ospfNbr') \
                      .get(columns=nbrcols, **kwargs)
         if nbr_df.empty:
+            missing_cols = set(ospfschema.get_display_fields(['default'])) - set(df.columns)
+            for col in missing_cols:
+                df[col] = np.nan
+            df = df.fillna({'peerIP': '-', 'peerHostname': '',
```

Comment:
Why fill peerIP with "-", and not "" like the others?

Reply:
I wanted to play it safe and comply with preexisting code. Take a look at row 130 of the same file:
```
df = df.drop(columns=list(df.filter(regex='_y$|_x$')),
                     errors='ignore') \
            .fillna({'peerIP': '-', 'numChanges': 0,
                     'lastChangeTime': 0})
```
A similar behavior can be observed within the bgp.py and lldp.py files in the same directory. However, I think that, unless there's specific code around which wants "-" instead of "" to filter out data for those columns, "" should be used everywhere because it makes it easier to detect that the field is, in fact, empty. That may require changes which don't directly target this issue though.